### PR TITLE
feat: introduce filters library

### PR DESCRIPTION
**Pull request by derberg**
**Description**

-  introducing filters library
- this library has not only some custom functions but also entire lodash
- release pipeline once it is approved

**Related issue(s)**
See also https://github.com/asyncapi/generator/issues/212

### CODE REVIEW

File: src/customFilters.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+const Markdown = require('markdown-it');
+const _ = require('lodash');
+const OpenAPISampler = require('openapi-sampler');
+
+const filter = module.exports;
+
+/**
+Filters that go away by adding loadash:
```
Comment (fmvilas) [inline src/customFilters.js:8]:
```suggestion
Filters that go away by adding lodash:
```

File: src/customFilters.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+/**
+ * Turns Markdown into HTML
+ * @md {string} - String with valid Markdown syntax
+ * @returns {string} HTML string
+ */
+function markdown2html(md) {
+  return Markdown().render(md || '');
+}
+filter.markdown2html = markdown2html;
```
Comment (fmvilas) [inline src/customFilters.js:32]: What about rewriting the filters like this:

```suggestion
filter.markdown2html = function(md) {
  return Markdown().render(md || '');
}
```

File: src/customFilters.js (hunk 1 above)
Comment (derberg) [reply src/customFilters.js:8]: ups, this is actually something that I should remove from PR as it was just a note for me :)

File: src/customFilters.js (hunk 2 above)
Comment (derberg) [reply src/customFilters.js:15]: yeah, but the problem is that once I do it this way `jsdoc2md` doesn't include this function in the docs

Comment (fmvilas) [reply src/customFilters.js:15]: Oh got it! Forget about it then 👍

**Pull request merged**